pkg/cmd/attestation/verification/tuf.go

```diff
@@ -32,11 +35,18 @@ func DefaultOptionsWithCacheSetting(tufMetadataDir o.Option[string]) *tuf.Option
 	// Allow TUF cache for 1 day
 	opts.CacheValidity = 1
 
+	// configure fetcher timeout and retry
+	f := fetcher.NewDefaultFetcher()
```

Comment:
@BagToad go-tuf is now imported as direct dependency because I call this `fetcher.NewDefaultFetcher` constructor from the library